### PR TITLE
Make config file optional

### PR DESCRIPTION
**Pull request by tbm**

### CODE REVIEW

**Comment by tbm** [comment]
Hmm, trusty doesn't have the new Perl module I need and Ubuntu Xenial is not available. Maybe we could instill the module with some other package manager?

**Comment by tbm** [comment]
"As you saw, we are working on dist: xenial. It may or may not work, depending on what we are working on." from https://github.com/travis-ci/travis-ci/issues/7260

**Comment by zacchiro** [comment]
so, the build currently fails at apt-get with:
```
E: Unable to locate package libcarp-assert-perl
E: Package 'libdate-calc-perl' has no installation candidate
E: Unable to locate package libconfig-onion-perl
E: Package 'libyaml-libyaml-perl' has no installation candidate
E: Package 'libfile-basedir-perl' has no installation candidate
```
which is *worse* than before, as at least cald/yaml/basedir modules were available. If we revert to a state where those are installable via apt, I believe we can work around the fact that onion is not available, using the cpan CLI installer.

**Comment by tbm** [comment]
Build passes with cpanm.

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ language: python
 python:
   - "3.6"
 install:
-  - sudo apt-get install libcarp-assert-perl libdate-calc-perl libyaml-libyaml-perl libfile-basedir-perl libexperimental-perl
+  - sudo apt-get install libcarp-assert-perl libdate-calc-perl libyaml-libyaml-perl libfile-basedir-perl libexperimental-perl cpanminus
+  - sudo cpanm Config::Onion
```
Comment (zacchiro) [inline .travis.yml]: *thumb up*

File: bin/ledger2beancount-accounts [hunk 2]
```diff
@@ -9,17 +9,20 @@ use strict;
...
-	$config = LoadFile($config_file);
-	last;
-    }
+    $config = Config::Onion->load($config_file);
+    last if defined $config;
 }
-die "No config file found (tried ", (join ", ", @config_files), ")\n" if !$config;
+
+$config->set_default(account_open_date => "1970-01-01");
+$config->set_default(beancount_indent => 2);
```
Comment (zacchiro) [inline bin/ledger2beancount-accounts]: it's of course annoying to have this stuff duplicated across multiple scripts, but i'm assuming we'll get rid of it with #5 , so it's fine

**Pull request merged**